featuretools/__main__.py

```diff
@@ -18,8 +19,12 @@ def info():
 
 @click.command(name='list-primitives')
 def list_primitives():
-    with pd.option_context('display.max_rows', None, 'display.max_columns', None, 'display.max_colwidth', None, 'display.width', 1000):
-        print(featuretools.list_primitives())
+    if version.parse(pd.__version__) < version.parse('1.0.0rc0'):
+        with pd.option_context('display.max_rows', None, 'display.max_columns', None, 'display.max_colwidth', -1, 'display.width', 1000):
```

Comment:
This can be simplified by making max_colwidth a variable that is either set to -1 or None